src/daemon/daemon_rpc.cpp

```diff
@@ -59,6 +62,11 @@ auto make_server(const std::string& server_address, multipass::Rpc::Service* ser
     if (server == nullptr)
         throw std::runtime_error("Failed to start multipass gRPC service at " + server_address);
 
+    auto group = getgrnam("adm");
```

Comment:
We need our own group here, like "multipass".

Reply:
I'm not really sure.  I was going by what @Saviq specified in the Trello card.

Reply:
OK, short term adm is fine.